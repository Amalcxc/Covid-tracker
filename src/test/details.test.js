import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import createStore from '../Redux/configureStore';
import Detail from './mock/details';

describe('component functionality', () => {
  test('component snapshot', () => {
    const tree = renderer.create(
      <Provider store={createStore}>
        <Detail />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('should render Mission component correctly', () => {
    const component = render(
      <Detail/>,
    );
    expect(component.container).toHaveTextContent('todays confirmed Cases');
  });
});