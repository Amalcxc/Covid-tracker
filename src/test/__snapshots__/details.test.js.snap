// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component functionality component snapshot 1`] = `
<div
  className="container p-4"
>
  <div
    className="row"
  >
    <div
      className="col-sm-4"
    />
    <div
      className="col-sm-8"
    >
      <h1
        className="text-white"
      >
        Loading
      </h1>
    </div>
  </div>
</div>
`;
